Assets/PatchKit Patcher/Scripts/Utilities/LauncherUtilities.cs

```diff
@@ -22,11 +22,11 @@ public static string LauncherExecutableByPlatform(PlatformType platform)
                 case PlatformType.OSX:
                     return "Launcher.app";
                 default:
-                    return "";
+                    throw new ArgumentException("Couldn't resolve launcher executable name.");
```

Comment:
I'd rather put a message like: "Unknown platform: " + platform. It's a developer mistake and the message should clearly tell what is wrong.